RevenueCatUI/PaywallView.swift

```diff
@@ -254,21 +255,47 @@ public struct PaywallView: View {
 
         #if PAYWALL_COMPONENTS
         if let paywallComponents = offering.paywallComponents {
-            PaywallsV2View(
-                paywallComponents: paywallComponents,
+
+            // For fallback view or footer
+            let paywall: PaywallData = .createDefault(with: offering.availablePackages,
+                                                      locale: self.locale)
+            let paywallView = LoadedOfferingPaywallView(
```

Comment:
Hmm do we have to build the V1 paywallView before checking if it's actually needed? I guess we would need to validate the data before displaying it which might be quite a refactor... So I'm ok with this for now.

Reply:
@tonidero Just updated this if you want to have another look again! I moved the creation the view to inside `PaywallsV2View` by passing an enum with data (either data for the `LoadedOfferingPaywallView` for production or `AnyView` for tests)

Reply:
Ah nice! Yeah I think that's an improvement! I guess we do need to create the data for the default paywall before, but that's already a big improvement 👍